Tools/MiniBrowser/wpe/main.cpp

```diff
@@ -418,6 +423,9 @@ static void activate(GApplication* application, WPEToolingBackends::ViewBackend*
         "user-content-manager", userContentManager,
         "is-controlled-by-automation", automationMode,
         "website-policies", defaultWebsitePolicies,
+#if ENABLE_WPE_PLATFORM
+        "display", wpeDisplay,
+#endif
```

Comment:
We can create the display before the web view and unref after it, since the view takes a reference of the display.

Reply:
Right, I'll update it.